doc/source/dev/development_environment.rst

```diff
@@ -91,12 +91,12 @@ It's possible to do a parallel build with ``numpy.distutils`` with the ``-j`` op
 see :ref:`parallel-builds` for more details.
 
 In order to install the development version of NumPy in ``site-packages``, use
-``python setup.py install --user``.
+``pip install . --user``.
 
 A similar approach to in-place builds and use of ``PYTHONPATH`` but outside the
 source tree is to use::
 
-    $ python setup.py install --prefix /some/owned/folder
+    $ pip install . --prefix /some/owned/folder
     $ export PYTHONPATH=/some/owned/folder/lib/python3.4/site-packages
```

Comment:
@rgommers should I remove those two sections then ? If it is not recommended to do so...

Reply:
Actually, I suggest to keep this line (with `pip install .`) because it's already marked as "other build option" so it's clear it's not one of the main options. 

The line _In order to install the development version of NumPy in site-packages, use pip install . --user._ needs removing though.

Reply:
removed.